### PR TITLE
[onkyo]  Add additional models

### LINKED ISSUE #1361: openhab2 - onkyo-binding - TX-NR509
With the onkyo-binding 1.9.0 works the TX-NR509 perfectly. With the onkyo-binding 2.0.0 will the receiver found as unsupported? Can you add the 509 to the devices in the binding?

thx
LN
## 

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/38777078-openhab2-onkyo-binding-tx-nr509?utm_campaign=plugin&utm_content=tracker%2F2164344&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F2164344&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by pail23**
- Add TX-NR509 and TX-NR555 as supported models
- Add additional Listen Modes

fixes #1361 and #1386

Signed-off-by: Paul Frank <pail@gmx.net>

### CODE REVIEW

**Comment by tavalin** [comment]
I have a TX-NR616 which shows up as an unsupported device but it mostly seems to work.
What's the strategy for adding supported models? The Input Source list on my 616 is different to some of the other models so should we be adding new thing-types?

File: addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -146,11 +146,23 @@
 		<state>
 			<options>
 				<option value="00">Stereo</option>
+                <option value="03">Game RPG</option>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml]: something is messed up here reg tabs vs spaces

File: addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/handler/OnkyoHandler.java [hunk 2]
```diff
@@ -264,83 +264,89 @@ public void statusUpdateReceived(EventObject event, String ip, String data) {
...
+                        updateState(CHANNEL_VOLUMEZONE2, new PercentType(Integer.parseInt(data.substring(3, 5), 16)));
+                        break;
+                    case ZONE2_SOURCE_SET:
+                        int inputZone2 = Integer.parseInt(data.substring(3, 5), 16);
+                        updateState(CHANNEL_INPUTZONE2, new DecimalType(inputZone2));
+                        break;
+                    default:
+                        logger.debug("Received unhandled status update from Onkyo Receiver @{}: data={}",
+                                connection.getConnectionName(), data);
+
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/handler/OnkyoHandler.java]: remove empty line here

**Comment by kaikreuzer** [comment]
That was quick, thanks!

**Pull request merged**